poucave/html/index.html

```diff
@@ -5,13 +5,16 @@
   <link rel="icon" href="" type="image/png" sizes="32x32"></link>
 </head>
 <body>
+  <object id="svg-diagram" data="../diagram.svg" type="image/svg+xml"></object>
```

Comment:
* just make this `/diagram.svg`?, it's served from the root anyways.  No need for a relative URL 
* Having this `<object>` says:  a diagram MUST exist.   A diagram should be optional.

Reply:
> Having this <object> says: a diagram MUST exist. A diagram should be optional.

I don't agree. It will just show a 404. Like a favicon or whatever.

Reply:
favicons should not 404 either. For this it probably doesn't matter as the traffic and impact will be quite small.